packages/fast-components-react-base/package.json

```diff
@@ -85,6 +86,7 @@
     "react-dom": "^16.3.0"
   },
   "dependencies": {
-    "@microsoft/fast-components-class-name-contracts-base": "^1.6.0"
+    "@microsoft/fast-components-class-name-contracts-base": "^1.6.0",
+    "uglifyjs-webpack-plugin": "^1.2.5"
```

Comment:
this doesn't seem right - why would a webpack plugin be a dependency?

Reply:
Should we dev-, I'll update

Reply:
And I don't think we need it there - this is a mistake.

Reply:
Updated!